src/components/DevTools/DevTools.anticov.ts

```diff
@@ -0,0 +1,97 @@
+  notifySuccess,
+  notifyFailure,
+} from '../../services/FeedbackService'
+
+const root = sdk.Identity.buildFromMnemonic(ANTICOV_CONFIG.ROOT_SEED)
+
+const ctype = sdk.CType.fromCType(ANTICOV_CONFIG.CTYPE as sdk.ICType)
+ctype.owner = root.address
+const metadata: IMetadata = ANTICOV_CONFIG.CTYPE_METADATA
+
+const delegationRoot = new sdk.DelegationRootNode(
+  ANTICOV_CONFIG.DELEGATION_ROOT_ID,
+  ctype.hash,
+  root.address
+)
+
+async function newDelegation(delegee: IMyIdentity): Promise<void> {
+  const delegationNode = new sdk.DelegationNode(
+    sdk.UUID.generate(),
+    delegationRoot.id,
+    delegee.identity.address,
+    [sdk.Permission.ATTEST]
+  )
+  const signature = delegee.identity.signStr(delegationNode.generateHash())
+  await delegationNode.store(root, signature)
+  notifySuccess(`Delegation successfully created for ${delegee.metaData.name}`)
+  await DelegationsService.importDelegation(
+    delegationNode.id,
+    'AntiCov Attester',
+    false
+  )
+  notifySuccess(`Delegation imported. Switch to Delegation Tab to see it.`)
+  const messageBody: sdk.MessageBody = {
+    type: sdk.MessageBodyType.INFORM_CREATE_DELEGATION,
+    content: { delegationId: delegationNode.id, isPCR: false },
+  }
+  await MessageRepository.sendToAddresses(
+    [delegee.identity.address],
+    messageBody
+  )
+}
+
+async function verifyOrAddCtypeAndRoot(): Promise<void> {
+  if (!(await ctype.verifyStored())) {
+    await ctype.store(root)
+    CTypeRepository.register({
+      cType: ctype,
+      metaData: { metadata, ctypeHash: ctype.hash },
+    })
+    notifySuccess(`CTYPE ${metadata.title.default} successfully created.`)
+  }
+  // delegationRoot.verify() is unreliable when using the currently released mashnet-node & sdk
+  // workaround is checking the ctype hash of the query result; it is 0x000... if it doesn't exist on chain
+  const queriedRoot = await sdk.DelegationRootNode.query(delegationRoot.id)
+  if (queriedRoot?.cTypeHash !== ctype.hash) {
+    await delegationRoot.store(root)
+    const messageBody: sdk.MessageBody = {
+      type: sdk.MessageBodyType.INFORM_CREATE_DELEGATION,
+      content: { delegationId: delegationRoot.id, isPCR: false },
+    }
+    notifySuccess(`AntiCov Delegation Root successfully created.`)
+    // sending root owner message for importing the root
+    const message = new sdk.Message(messageBody, root, root)
+    await MessageRepository.dispatchMessage(message)
+    notifySuccess(`Sent Delegation Root to AntiCov root authority.`)
+  }
+}
+
+export async function setupAndDelegate(delegee: IMyIdentity): Promise<void> {
+  const blockUi = FeedbackService.addBlockUi({
+    headline: 'Creating AntiCov Delegation',
+  })
+  try {
+    blockUi.updateMessage('transferring funds to AntiCov authority')
+    await new Promise(resolve => {
+      BalanceUtilities.makeTransfer(delegee, root.address, 4, () => resolve())
+    })
+    blockUi.updateMessage('setting up CType and Root Delegation')
```

Comment:
_As above_: To make this consistent with the rest of the UI notifications, we should start with a capital letter (but not convert to a sentence).